qa/workunits/rados/test_rados_tool.sh

```diff
@@ -591,5 +567,10 @@ test_cleanup
 test_append
 test_put
 
+# clean up environment, delete pool
+$CEPH_TOOL osd pool delete $POOL $POOL --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_EC $POOL_EC --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_CP_TARGE $POOL_CP_TARGET --yes-i-really-really-mean-it
```

Comment:
ok, i  have tried it.
ceph returns 0 if pool isn't exist.
ceph returns 1 if pool1 != pool2 and pool1 exists.